tests/serverpod_test_server/lib/src/protocol/entities_with_relations/many_to_many/enrollment.yaml

```diff
@@ -1,8 +1,8 @@
 class: Enrollment
 table: enrollment
 fields:
-  student: Student?, relation(name=student_enrollments)
-  course: Course?, relation(name=course_enrollments)
+  student: Student?, relation(name=student_enrollments, onDelete=Cascade)
+  course: Course?, relation(name=course_enrollments, onDelete=Cascade)
```

Comment:
Question - Are the addition of these constraints used in any way? Would it be fine to not have them here so that the definitions are as simple as possible to explain the different configurable relations? I'm worried this might be confusing to readers.

Reply:
Yes because we have tests that are deleting things and checking the response, but they fail without this config as there are relations setup.

Reply:
Aaah, I understand 👍 

Maybe it makes sense to have a onDelete policy in place for these tests as well then since users probably will run into the same issues.